apps/backend/src/core/port/infrastructure/web/port.controller.ts

```diff
@@ -32,4 +32,9 @@ export class PortController {
   async listPorts(): Promise<any> {
     return this.portRepository.list();
   }
+
+  @Get("/listCountries")
+  async listCountries(): Promise<string[]> {
+    return this.portRepository.listCountries();
```

Comment:
**suggestion:** There was kind of a trick here, to have proper clean-archi segregation, it is better to not directly use your infra layer in your web layer. Instead should create a application/query that make use of underlaying adapters, and not expose them directly to controller